src/components/manage/Blocks/Search/SearchBlockView.jsx

```diff
@@ -89,6 +89,7 @@ const SearchBlockView = (props) => {
         setSelectedView={setSelectedView}
       >
         <ListingBody
+          id={id}
```

Comment:
@ksuess For the most part I agree with your analysis that the change is not breaking, but I think there is a possibility for it to break here if a project has customized the SearchBlockView. Then the ListingBody would not receive the `id` unless the customization is updated.

This could be handled in withQuerystringResults by making it use the `id` if present, otherwise fall back to the old approach (`data.block`).

Reply:
Updated.
https://github.com/plone/volto/pull/4239/commits/fec8fea12cbec0a65b99d90e23a662c79b0ee2ef